### PR TITLE
Add import/export of tags as bulk actions

### LINKED ISSUE #2056: Add new Export Tags to CSV bulk action
It should have two textareas:

1. A list of druids, populated from:
    - [ ] Search session (via JavaScript, like is done in the bulk update)
    - [ ] Paste in list
2. One or more tags

Each object and all of its tags will be exported to a CSV file for download

### LINKED ISSUE #2057: Add new Import Tags from CSV bulk action
It should have an "Upload a csv file" box (see the Create virtual objects bulk action)

### PR DESCRIPTION
**Pull request by mjgiarlo**
Fixes #2056
Fixes #2057

## Why was this change made?

This commit adds the abilities to import and to export tags in CSV form as asynchronous bulk actions, implementing the designs as specified.

Also includes:
* Clean up/repurpose prior "SetTags" work
* Regenerate rubocop TODOs
* Clarify in README and CI config that Ruby 2.7 is expected
* Fix the Redis URL in docker-compose configuration
  * This is looked up via the `config` gem, in the `Settings` constant now. I needed this change to test things out locally, to get the web container's Sidekiq connecting to Redis.
  * Also, sort the env variables that I touched to make it easier to see what is there.

## How was this change tested?

Tested locally. Will be tested in stage by @andrewjbtw 

## Which documentation and/or configurations were updated?

README, and a new config was added. (No shared_configs changes needed.)

### CODE REVIEW

File: app/jobs/export_tags_job.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+          log_buffer.puts("#{Time.current} #{self.class}: Exporting tags for #{druid} (bulk_action.id=#{bulk_action_id})")
+          csv << [druid, *export_tags(druid)]
+          bulk_action.increment(:druid_count_success).save
+        rescue StandardError => e
+          log_buffer.puts("#{Time.current} #{self.class}: Unexpected error exporting tags for #{druid} (bulk_action.id=#{bulk_action.id}): #{e}")
+          bulk_action.increment(:druid_count_fail).save
+        end
+      end
+
+      File.open(csv_download_path, 'w') { |file| file.write(csv_string) } if csv_string.present?
```
Comment (justinlittman) [inline app/jobs/export_tags_job.rb:26]: Why are we writing the CSV to memory instead of straight to file?

Comment (mjgiarlo) [reply app/jobs/export_tags_job.rb:26]: Damn good question. I'll make that change.

Comment (mjgiarlo) [reply app/jobs/export_tags_job.rb]: @justinlittman take another look? thanks for the suggestion!

**Comment by justinlittman** [comment]
Looks good @mjgiarlo !

**Comment by mjgiarlo** [comment]
I believe the Travis failure above is a transient one, so I am merging. Thanks, @justinlittman

**Pull request merged**